### DISCUSSION TITLE
Underwater Drone Hide and Seek (Project 27)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#5] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #27](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Underwater%20Drone%20Hide%20and%20Seek)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by richardsunny24** [comment]
No discussions on this project yet?

**Reply by robertogl** [reply]
Hi Richard,

Not yet, but apparently we have just started one! Do not hesitate to ask any question, we will be here to reply.

**Comment by Egarc528** [comment]
What is the Simulink model that we need to modify for this project?

**Reply by robertogl** [reply]
Hi @Egarc528,

We do not have a Simulink model that can 'just' be modified. However, you may find useful using an existing demo with an AUV model as a starting point. This is the repo of that [reference AUV application](https://github.com/mathworks-robotics/modeling-and-simulation-of-an-AUV-in-Simulink). And this is the related [tutorial video](https://www.mathworks.com/videos/design-modeling-and-simulation-of-autonomous-underwater-vehicles-1619636864529.html). Also, for questions related to the reference application, here are a list of [FAQs](https://www.mathworks.com/videos/series/auv-deep-dive.html).
I hope you will find it useful.

**Comment by evadrake89** [comment]
Hi, I am Aditya from BS Electronic Systems, IIT Madras and I am interested in working and collaborating on this project.......

**Reply by shiva2307** [reply]
> Hi, I am Aditya from BS Electronic Systems, IIT Madras and I am interested in working and collaborating on this project.......

can we work together on this project?
